### ISSUE TITLE
Dedicated overlay editor (Android)

### ISSUE THREAD

**Issue by Loop312** [Lime3DS/Lime-3DS-Emulator#101]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

A dedicated overlay controls editor for Android that can be used to easily modify the controls, things like haptic strength, individual button size, and opacity

### Why would this feature be useful?

Brings better customization to the overlay and gives a better user experience

**Comment by OpenSauce04** [comment]
The latter of the two things you mentioned are already features as of the last update

**Comment by Loop312** [comment]
Yeah I know, I was just saying that it would be nice to have a dedicated editor for the controls where you can edit individual buttons/dpad/sticks because currently doing it through the menu is a little tedious and finicky. Something like skyline had is what im thinking. I'm also willing to help with this if I can.

**Comment by OpenSauce04** [comment]
Something which could be useful is being able to use grid snapping when rearranging buttons. That would make aligning the buttons significantly easier
